### PR TITLE
Update Project.toml

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by schillic** [comment]
A minor release for adding a method?

File: Project.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = "IntervalMatrices"
 uuid = "5c1f47dc-42dd-5697-8aaa-4d102d140ba9"
-version = "0.2.2"
+version = "0.3.0"
```
Comment (mforets) [inline Project.toml]:
```suggestion
version = "0.2.3"
```

**Comment by mforets** [comment]
For reference, the distinction between patch and minor releases adopted in the Julia community can be found [here](https://discourse.julialang.org/t/proposed-release-process-and-schedule/15623), and this convention is [encouraged](https://discourse.julialang.org/t/proposed-release-process-and-schedule/15623/5) in Julia packages.

In [this release](https://github.com/JuliaReach/IntervalMatrices.jl/wiki/Release-log-tracker) we had incuded the PR in the "features" list so by default i mark these as minor releases, without further considerataion. However i agree that this particular method can better be seen as bug fix for [this PR](https://github.com/JuliaReach/Reachability.jl/pull/688/files#diff-9952ff8bb5cab26a32f4f893e6077c30R17).

**Comment by schillic** [comment]
First: As was pointed out in the link you gave, determining the version is subjective to a degree, so in general there is no clear algorithm.

> In particular, patch releases only contain docs updates/fixes and bug fixes

To me, the description in your link seems to imply that the main characteristic of a patch release is that it does not add breaking changes. Technically, fixing a bug is a breaking change, though, so this definition is again not precise. A better formulation could be: "Under usual circumstances, updating to a patch release should be without risk, while updating to a minor release has the risk of breaking some parts of your code."
Applying this definition here, adding a new method has very low risk of breaking things.

> In this release we had incuded the PR in the "features" list so by default i mark these as minor releases, without further consideration.

With my understanding from above, you would check the "breaking" list instead (assuming we filled it correctly).

**Pull request merged**